content/learn/book/migration-guides/0.5-0.6/_index.md

```diff
@@ -0,0 +1,239 @@
+}
+```
+
+### "Light" and "LightBundle" are now "PointLight" and "PointLightBundle"
+
+```rust
+// 0.5
+commands.spawn_bundle(LightBundle {
+    light: Light {
+        color: Color::rgb(1.0, 1.0, 1.0),
+        depth: 0.1..50.0,
+        fov: f32::to_radians(60.0),
+        intensity: 200.0,
+        range: 20.0,
+    },
+    ..Default::default()
+});
+
+// 0.6
+commands.spawn_bundle(PointLightBundle {
+    light: PointLight {
+        color: Color::rgb(1.0, 1.0, 1.0),
+        intensity: 200.0,
+        range: 20.0,
+    },
+    ..Default::default()
+});
+```
+
+The {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.5.0" name="Light" no_mod=true)}} and {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.5.0" name="LightBundle" no_mod=true)}} were renamed to {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.6.0" name="PointLight" no_mod=true)}} and {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.6.0" name="PointLightBundle" no_mod=true)}} to more clearly communicate the behavior of the Light Source.
+At the same time the `fov` and `depth` fields were removed from the {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.6.0" name="PointLight" no_mod=true)}} as they were unused.
+
+<!-- TODO: Remove this comment if https://github.com/bevyengine/bevy/pull/2367 is merged.
+
+In addition the {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.6.0" name="DirectionalLight" no_mod=true)}} and {{rust_type(type="struct" crate="bevy_pbr" mod="" version="0.6.0" name="DirectionalLightBundle" no_mod=true)}} were introduced with `0.6`.
+
+-->
+
+### System Param Lifetime Split
+
+The Lifetime of {{rust_type(type="trait" crate="bevy_ecs" mod="system" version="0.5.0" name="SystemParam" no_mod=true)}} was split in two separate Lifetimes.
+
+```rust
+// 0.5
+type SystemParamAlias<'a> = (Res<'a, AssetServer>, Query<'a, &'static Transform>, Local<'a, i32>);
+
+#[derive(SystemParam)]
+struct SystemParamDerive<'a> {
+    res: Res<'a, AssetServer>,
+    query: Query<'a, &Transform>,
+    local: Local<'a, i32>,
+}
+
+// 0.6
+type SystemParamAlias<'w, 's> = (Res<'w, AssetServer>, Query<'w, 's, &'static Transform>, Local<'s, i32>);
+
+#[derive(SystemParam)]
+struct SystemParamDerive<'w, 's> {
+    res: Res<'w, AssetServer>,
+    query: Query<'w, 's, &'static Transform>,
+    local: Local<'s, i32>,
+}
+```
+
+### QuerySet declare "QueryState" instead of "Query"
+<!-- Adapt for ParamSet instead, if https://github.com/bevyengine/bevy/pull/2765 is merged -->
+
+Due to the [System Param Lifetime Split](#system-param-lifetime-split), {{rust_type(type="struct" crate="bevy_ecs" mod="system" name="QuerySet" version="0.6.0" no_mod=true plural=true)}} now need to specify their Queries with {{rust_type(type="struct" crate="bevy_ecs" mod="query" version="0.6.0" name="QuerySet" no_mod=true)}} instead of {{rust_type(type="struct" crate="bevy_ecs" mod="system" version="0.6.0" name="Query" no_mod=true)}}.
```

Comment:
There's a typo here it says:

Due to the System Param Lifetime Split, QuerySets now need to specify their Queries with **QuerySet** instead of Query .

It should say:
"... Queries with **QueryState** instead of Query."